### PR TITLE
Added Text translation in Urdu

### PR DESCRIPTION
**Pull request by sperveen4**
Translated the questions and year selector in Urdu for the pie chart

### CODE REVIEW

File: app/components/questions/TrafficIntensity.js [hunk 1]
```diff
@@ -26,6 +26,26 @@ const CITY_PART_COLOR = {
   "West Side": "#882255",
 };
 
+const TEXT_TRANSLATION = {
+  "English": {
+  "Title": "Traffic Intensity by Area",
```
Comment (vingkan) [inline app/components/questions/TrafficIntensity.js:31]: You can indent the translation parts one more level, that way they are not on the same indentation level as the languages.

File: app/components/questions/TrafficIntensity.js [hunk 2]
```diff
@@ -99,6 +120,26 @@ function YearSelector(props) {
...
+        {props.data.map((d, i) => (
+          <option key={i} value={d}>{d}</option>
+        ))}
+      </select>
+    </div>
+  );
+}
 function ExamplePieChart(props) {
   const { data } = props
   console.log("examplepiechart",data)
```
Comment (vingkan) [inline app/components/questions/TrafficIntensity.js:145]: Can you delete this console.log and the one on line 195 that is commented out?

File: app/components/questions/TrafficIntensity.js (hunk 1 above)
Comment (vingkan) [reply app/components/questions/TrafficIntensity.js:31]: Actually, this is a code style comment and doesn't need to be blocking.

File: app/components/questions/TrafficIntensity.js (hunk 2 above)
Comment (vingkan) [reply app/components/questions/TrafficIntensity.js:145]: Actually, this is a cleanup comment and doesn't need to be blocking.

**Review by vingkan — APPROVED** [review]
Very exciting, great work Shahzia! I left two small comments, those are optional to fix.

I approved the PR so you can merge right away or merge after you fix the optional comments.

I realized there is no easy way to add translations for the title and description since that is handled outside this page.

Also tagging @alkatoutl who I think will be interested in this PR!

**Comment by vingkan** [comment]
Screenshot of the page with Urdu selected:

Screen Shot 2021-08-15 at 6 09 26 PM

**Review by Andrew-Tolentino — APPROVED** [review]

**Pull request merged**